newIDE/app/src/stories/index.js

```diff
@@ -2471,12 +2504,15 @@ storiesOf('InstructionEditor', module)
         objectsContainer={testProject.testLayout}
         isCondition
         instruction={testProject.testInstruction}
-        resourceExternalEditors={fakeResourceExternalEditors}
-        onChooseResource={() => {
-          action('onChooseResource');
-          return Promise.reject();
+        resourceManagementProps={{
```

Comment:
not to be a pain, but maybe this is the opportunity to move all those component stories into their own folder 👀 👀 👀 😬